crates/rustc_codegen_spirv/src/linker/specializer.rs

```diff
@@ -2005,7 +2005,8 @@ impl<'a, S: Specialization> InferCx<'a, S> {
             }
 
             if let Err(e) = self.equate_match_findings(m) {
-                e.report(inst);
+                // temporarily disabled, arrays of buffer descriptors cause errors here
+                // e.report(inst);
```

Comment:
I cannot figure out why the specializer is upset about my changes (or what exactly it does in the first place). For now I've just disabled the error reporting, and the spirv emitted looks fine. So I assume this is a bug / missing feature in the specializer rather than my PR?